pytest.ini

```diff
@@ -7,3 +7,4 @@ filterwarnings =
     ignore:the matrix subclass is not
     ignore:Using a non-tuple
     ignore:Input graph is not fully connected.
+    ignore:Number of distinct clusters
```

Comment:
what is this from

Reply:
This is from GMM. Realized there is no randomness in my tests so changed the undirected version to the one in the paper, but changed the probability values for directed case to be something different than 1. 

Removed this line.